webpack/components/AnsibleHostDetail/components/RolesTab/EditRolesModal/EditRolesForm.js

```diff
@@ -50,7 +51,7 @@ const EditRolesForm = props => {
       key="confirm"
       variant="primary"
       onClick={() => callMutation({ variables })}
-      isDisabled={loading}
+      isDisabled={loading || isEqual(initAvailableOptions, availableOptions)}
```

Comment:
For me button is still enabled, steps to reproduce:
* (host without any assigned roles)
* Show modal => button is disabled :heavy_check_mark: 
* Select role(s) => button is enabled :heavy_check_mark: 
* Remove all assigned roles => button is enabled  :x:

Reply:
Yop, here it is:

https://user-images.githubusercontent.com/59385976/176603099-8252d8c8-a26a-47da-a024-8d035dbea97a.mp4